core/src/main/resources/db/migration.sql

```diff
@@ -22,3 +22,5 @@ PREPARE stmt FROM @s;
 EXECUTE stmt;
 DEALLOCATE PREPARE stmt;
 UPDATE info SET DB_SCHEMA_VERSION="1.2.1";
+
+
```

Comment:
Looks like you are missing an entry for new comments, schema version, and the SV table creation.